### ISSUE TITLE
Cannot read property 'getAggregateSourceElements' of undefined

### ISSUE THREAD

**Issue by reda-benhemmouche** [forcedotcom/cli#256]
### Summary
I get error without details during the execution of the command `sfdx force:source:convert`

### Steps To Reproduce:

1.  This is step 1.
run the command `sfdx force:source:convert`
1.  This is step 2. 
get error 
`ERROR running force:source:convert:  Cannot read property 'getAggregateSourceElements' of undefined`

### Expected result

Sometime the error is related to incorrect folder name of non-supported file extension.
We need the details about the metadatas (file, folders) that are  causing this error to be able to fix it.

### Actual result

No details about the metadata causing this error.
```

$ sfdx force:source:convert -d ./src/ --json
{
    "status": 1,
    "name": "TypeError",
    "message": "Cannot read property 'getAggregateSourceElements' of undefined",
    "exitCode": 1,
    "commandName": "SourceConvertCommand",
    "stack": "TypeError: Cannot read property 'getAggregateSourceElements' of undefined\n    at initWorkspaceAdapter.then.catch.then (/Users/redabenhemmouche/.local/share/sfdx/client/7.36.0-b1c4e9c7c8/node_modules/salesforce-alm/dist/lib/source/sourceConvertApi.js:73:52)\nOuter stack:\n    at Function.wrap (/Users/redabenhemmouche/.local/share/sfdx/client/7.36.0-b1c4e9c7c8/node_modules/@salesforce/core/lib/sfdxError.js:151:27)\n    at SourceConvertCommand.catch (/Users/redabenhemmouche/.local/share/sfdx/client/7.36.0-b1c4e9c7c8/node_modules/salesforce-alm/dist/ToolbeltCommand.js:247:46)",
    "warnings": []
}

```

### Additional information

```

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
`sfdx-cli/7.36.0-b1c4e9c7c8 darwin-x64 node-v10.15.3`
**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.1 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.8 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.5 (core)
analytics 1.2.1 (core)
generator 1.1.1 (core)
salesforcedx 47.9.0 (core)
├─ salesforcedx-templates 47.9.0 (core)
└─ salesforce-alm 47.10.0 (core)

sfdx-cli 7.36.0 (core)
sfdx-essentials 1.0.3
sfpowerkit 1.30.1

```
**OS and version**:
mac os high sierra
version: 10.13.6 (17G8037)

**Comment by pramodgavade** [comment]
@reda-benhemmouche - were you able to find the root cause? I am also getting same error while using Jenkins. Strangely it works if I use VSCode.

**Comment by pramodgavade** [comment]
@reda-benhemmouche - With some trial and error, I found out the root cause; "lwc" metadata is causing this issue. It works if I remove lwc folder.

**Comment by kiritgarg** [comment]
Did you get any solution to this? I have removed LWC folder. Still i am getting same error.

**Comment by shetzel** [comment]
@reda-benhemmouche - thank you for posting an excellent bug report with the stack trace!  I see the problem in the code and will fix asap.  I would expect the fix in the next patch release of the core plugin.

**Comment by MukundKoiralaDH** [comment]
@shetzel is there any planned date to release this fix ?

**Comment by shetzel** [comment]
The next patch release, which is this Thursday.  To be clear, the error message reported in the issue will no longer be thrown.  Instead, the actual problem that caused the error will be thrown, which should provide the context for people hitting this issue.

**Comment by Gteixeirinha7** [comment]
Hi, i am today running a Continous Integration in my orgs, found the same problem with the component ApexEmailNotification, as a solution i remove this folder from the project

Hope help someone this information

**Comment by shetzel** [comment]
@Gteixeirinha7 - what is the output of `sfdx plugins --core`?  If you are on the latest core plugins you should not see this error.

**Comment by Gteixeirinha7** [comment]
Today i am using a pipeline running in Bitbucket with this docker image: https://hub.docker.com/r/salesforce/salesforcedx

the output of `sfdx plugins --core` is: 
```

+ sfdx plugins --core
 ›   Warning: You acknowledge and agree that the CLI tool may collect usage 
 ›   information, user environment, and crash reports for the purposes of 
 ›   providing services or functions that are relevant to use of the CLI tool 
 ›   and product improvements.
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.1 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.8 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-trust 3.0.5 (core)
analytics 1.2.1 (core)
generator 1.1.1 (core)
salesforcedx 47.9.0 (core)
├─ salesforcedx-templates 47.9.0 (core)
└─ salesforce-alm 47.10.0 (core)
sfdx-cli 7.36.0 (core)
```

**Comment by shetzel** [comment]
That is a really old version.  Should be 7.74.1 for the sfdx-cli version.  https://hub.docker.com/r/salesforce/salesforcedx/tags
